plugins/org.komodo.core/src/config/vdb.cnd

```diff
@@ -107,6 +107,7 @@
 
 // permission resource path is node path
 [vdb:permission] > nt:unstructured
+ - vdb:resourceName (string) mandatory
```

Comment:
OH!!!

That never occurred to me when I was handling them. Right shall fix that asap and repost the commit.